web/cypress/integration/fire-behaviour-advisory-calculator-page.spec.ts

```diff
@@ -2,6 +2,30 @@ import { FIRE_BEHAVIOR_CALC_ROUTE } from '../../src/utils/constants'
 import { FuelTypes } from '../../src/features/fbaCalculator/fuelTypes'
 import { DateTime } from 'luxon'
 
+const datePickerTest = (setDate: Function, visitAndAddRow: Function) => {
```

Comment:
It looks like these tests do stuff but aren't asserting anything, do we need to expect certain results?

Reply:
`cy.intercept('POST', 'api/fba-calc/stations', req => {
    expect(req.body).to.deep.include({
      date: yesterday
    })` It happens here from what I could tell, I just repurposed a test that was already here.

Reply:
We need logical asserts for these tests to mean anything.

Reply:
My bad, I didn't see the expect assertions in the request body intercept